rust/protocol/src/storage/traits.rs

```diff
@@ -94,8 +145,10 @@ pub trait SessionStore {
     ) -> Result<()>;
 }
 
+/// Interface for storing sender key records for other users, allowing multiple keys per user.
 #[async_trait(?Send)]
 pub trait SenderKeyStore {
+    /// Set the entry for `(sender, distribution_id)` to the value of `record`.
```

Comment:
"the value of" is kind of redundant. An in-memory store might save the object as is (cloning it); a database will probably serialize it. And I'm realizing now that it should probably pass the record by value for this reason, but let's do that separately.

Reply:
I've shuffled this sentence to say "Assign `record` to the entry for..."!